nvmf-autoconnect/systemd/nvmf-connect-nbft.service.in

```diff
@@ -0,0 +1,11 @@
+# This unit is meant to be started by network management software
+# after a network interface defined in the NBFT gets set up
+[Unit]
+Description=Connect NBFT-defined NVMe-oF subsystems automatically
+ConditionPathExists=|/sys/firmware/acpi/tables/NBFT
+ConditionPathExists=|/sys/firmware/acpi/tables/NBFT1
+
+[Service]
+Type=oneshot
+ExecStartPre=/sbin/modprobe nvme-fabrics
```

Comment:
Instead of using `ExecStartPre` to load the kernel module, I think that the preferred way is to do as follows under the `[Unit]` section:
```
[Unit]

Wants=modprobe@nvme_fabrics.service
After=modprobe@nvme_fabrics.service
```

Reply:
Sounds reasonable. Good catch! Thanks

Reply:
Call me stupid, but where are the actual 'connect' calls?
Are they done by the 'nvmf-autoconnect.service'?
If so, shouldn't we add an explicit order like 'Before' here?
If not, how does autoconnect happen?

Reply:
And, incidentally, we're using 'ExecStartPre' in 'nvmf-autoconnect.in', too. Maybe we should be changing that, too.

Reply:
> Call me stupid, but where are the actual 'connect' calls? Are they done by the 'nvmf-autoconnect.service'? If so, shouldn't we add an explicit order like 'Before' here? If not, how does autoconnect happen?

The nvm connect-all --nbft call is currently done by dracut. 

https://github.com/dracutdevs/dracut/blob/6acfecae572fb457115b276b5b64d9424ad5187b/modules.d/95nvmf/nvmf-autoconnect.sh#L35

So maybe it's not a good idea to add it here.